Bottom: d0827885742f4354c2dc8176ce267c429ec766d0
Top:    5e93d68409472371562e2cc7dd048f9952cc7282
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-21 14:19:47 +1300

Refresh of threadsafestreamresult

---

diff --git a/testtools/tests/test_testsuite.py b/testtools/tests/test_testsuite.py
index 163889c..e466080 100644
--- a/testtools/tests/test_testsuite.py
+++ b/testtools/tests/test_testsuite.py
@@ -163,7 +163,7 @@ class TestConcurrentStreamTestSuiteRun(TestCase):
         # Check the traceback loosely.
         self.assertThat(events[1][2].decode('utf8'), DocTestMatches("""\
 Traceback (most recent call last):
-  File "...testtools/testsuite.py", line 181, in _run_test
+  File "...testtools/testsuite.py", line 188, in _run_test
     test.run(process_result)
 TypeError: run() takes exactly 1 ...argument (2 given)
 """, doctest.ELLIPSIS))
